scss/4-elements/_buttons.scss

```diff
@@ -67,6 +67,13 @@ $button-shadow-size: 4px;
   &:active {
     border-color: $cads-language__primary-button-hover-colour;
   }
+
+  .cads-icon {
+    fill: currentColor;
```

Comment:
Yeah I think this is much more sensible - let's replace the default one in https://github.com/citizensadvice/design-system/blob/4100d28eea4d17f72cccc114922003579d0e63e7/scss/6-components/_icons.scss#L8 and associated settings